src/status_im/react_native/resources.cljs

```diff
@@ -69,7 +69,8 @@
    :hardwallet-card     (js/require "./resources/images/ui/hardwallet-card.png")
    :secret-keys         (js/require "./resources/images/ui/secret-keys.png")
    :keycard-lock        (js/require "./resources/images/ui/keycard-lock.png")
-   :hold-card-animation (js/require "./resources/images/ui/hold-card-animation.png")
+   :hold-card-animation (try (js/require "./resources/images/ui/hold-card-animation.gif")
```

Comment:
Why?

Reply:
because `learn test-cljs` fails as `.gif` file can't be required. Why? Have no idea. 
That file originally was named as `something.png` which is definitely not the case, and it worked fine but after upgrade android refuses to put invalid png into resources.